src/easy-tab-accordion.js

```diff
@@ -134,6 +134,7 @@ export class EasyTabAccordion{
         this.hasInitialized = false;
         this.enabled = validBreakpoints(this) ? isLive(this) : true;
         this.count = this.wrapper.querySelectorAll(this.options.trigger).length;
+        this.isFirst = true;
```

Comment:
What does this `first` stand for? Please put a comment here.

Reply:
After loading the page, the script will check 2 situations:
- A hash on the address bar
- Default active section

By default, the script will check this id (generate from hash/activeSection) and invoke openPanel(id). Inside that function, after opening the section (just an animation because all the sections are opened by default), the script will call this code.

![image](https://user-images.githubusercontent.com/30406982/193797879-dd62d184-8be5-43e0-9b2a-98f52e7f47df.png)

But no sections are **active**. So I have added a small condition for checking it when init.